packages/fluentui/react-northstar/src/components/SplitButton/SplitButtonToggle.tsx

```diff
@@ -0,0 +1,174 @@
+}
+
+export type SplitButtonToggleStylesProps = Pick<SplitButtonToggleProps, 'primary' | 'disabled'>;
+
+const SplitButtonToggle: React.FC<WithAsProp<SplitButtonToggleProps>> &
+  FluentComponentStaticProps<SplitButtonToggleProps> = props => {
+  const context: ProviderContextPrepared = React.useContext(ThemeContext);
+  const { setStart, setEnd } = useTelemetry(SplitButtonToggle.displayName, context.telemetry);
+  setStart();
+
+  const {
+    accessibility,
+    // @ts-ignore
+    active,
+    as,
+    children,
+    content,
+    disabled,
+    primary,
+    className,
+    styles,
+    variables,
+    design,
+  } = props;
+
+  const hasChildren = childrenExist(children);
+
+  const getA11Props = useAccessibility(accessibility, {
+    debugName: SplitButtonToggle.displayName,
+    mapPropsToBehavior: () => ({
+      as,
+      active,
+      disabled,
+    }),
+    actionHandlers: {
+      performClick: event => {
+        event.preventDefault();
+        handleClick(event);
+      },
+    },
+    rtl: context.rtl,
+  });
+  const { classes } = useStyles<SplitButtonToggleStylesProps>(SplitButtonToggle.displayName, {
+    className: SplitButtonToggle.className,
+    mapPropsToStyles: () => ({
+      primary,
+      disabled,
+    }),
+    mapPropsToInlineStyles: () => ({
+      className,
+      design,
+      styles,
+      variables,
+    }),
+    rtl: context.rtl,
+  });
+
+  const unhandledProps = useUnhandledProps(SplitButtonToggle.handledProps, props);
+  const ElementType = getElementType(props);
+
+  const handleClick = (e: React.SyntheticEvent) => {
+    if (disabled) {
+      e.preventDefault();
+      return;
+    }
+
+    _.invoke(props, 'onClick', e, props);
+  };
+
+  const handleFocus = (e: React.SyntheticEvent) => {
+    _.invoke(props, 'onFocus', e, props);
+  };
+
+  const result = (
+    <ElementType
+      {...rtlTextContainer.getAttributes({ forElements: [children] })}
+      {...getA11Props('root', {
+        onClick: handleClick,
+        disabled,
+        className: classes.root,
+        onFocus: handleFocus,
+        ...unhandledProps,
+      })}
+    >
+      {hasChildren ? children : content}
+    </ElementType>
+  );
+
+  setEnd();
+
+  return result;
+};
+
+SplitButtonToggle.defaultProps = {
+  as: 'button',
+  accessibility: buttonBehavior,
+};
+
+SplitButtonToggle.displayName = 'SplitButtonToggle';
+SplitButtonToggle.className = 'ui-splitbutton__toggle';
+
+SplitButtonToggle.propTypes = {
+  ...commonPropTypes.createCommon({}),
+  disabled: PropTypes.bool,
+  onClick: PropTypes.func,
+  onFocus: PropTypes.func,
+  primary: customPropTypes.every([customPropTypes.disallow(['secondary']), PropTypes.bool]),
+  secondary: customPropTypes.every([customPropTypes.disallow(['primary']), PropTypes.bool]),
+};
+
+SplitButtonToggle.handledProps = Object.keys(SplitButtonToggle.propTypes) as any;
+
+SplitButtonToggle.create = createShorthandFactory({ Component: SplitButtonToggle, mappedProp: 'children' });
```

Comment:
```suggestion
SplitButtonToggle.create = createShorthandFactory({ Component: SplitButtonToggle, mappedProp: 'content' });
```

I think that it's `content`